include/z64item.h

```diff
@@ -377,6 +377,127 @@ typedef enum {
     /* 0x7E */ GI_MAX
 } GetItemID;
 
+typedef enum {
+    /* 0x00 */ DI_BOTTLE,
+    /* 0x01 */ DI_KEY_SMALL,
+    /* 0x02 */ DI_SONG_MINUET,      // unused
+    /* 0x03 */ DI_SONG_BOLERO,      // unused
+    /* 0x04 */ DI_SONG_SERENADE,    // unused
+    /* 0x05 */ DI_SONG_REQUIEM,     // unused
+    /* 0x06 */ DI_SONG_NOCTURNE,    // unused
+    /* 0x07 */ DI_SONG_PRELUDE,     // unused
```

Comment:
I personally think the unused comments are unnecessary, and they could be misunderstood as "not implemented"